### PR TITLE
feat(jdbc): Add MariaDB support 

### PR DESCRIPTION
**Pull request by johnBgood**
## Description

- Add support for MariaDB
- Fix password encoding issue
- Add the jdbc connector to the bundle
- Fix and reworked integration tests to run a MariaDB container

## Related issues

closes https://github.com/camunda/team-connectors/issues/740
closes https://github.com/camunda/team-connectors/issues/773

### CODE REVIEW

File: bundle/default-bundle/pom.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (ev-codes) [inline bundle/default-bundle/pom.xml:1]: **(nitpicky) Question:** Why does the XML namespace change here?

I wouldn't expect it to change based on the changes elsewhere in the PR.

**Review by ev-codes — COMMENTED** [review]
Looks great! Just one small question 😄

File: bundle/default-bundle/pom.xml (hunk 1 above)
Comment (johnBgood) [reply bundle/default-bundle/pom.xml]: It's coming from my IntelliJ's reordering config. I changed it thanks

File: bundle/pom.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (ev-codes) [inline bundle/pom.xml:1]: XML namespace here too :smile:

**Review by ev-codes — APPROVED** [review]
One more namespace change to adjust 😄

File: bundle/pom.xml (hunk 2 above)
Comment (johnBgood) [reply bundle/pom.xml]: now we're good to go I guess 💯

**Pull request merged**